DESCRIPTION

```diff
@@ -15,7 +15,16 @@ Authors@R: c(
     person("Merck Sharp & Dohme Corp", role = c("cph"))
     )
 Description: A metadata structure for clinical data analysis
-    and reporting based on Analysis Data Model (ADaM) datasets.
+    and reporting based on Analysis Data Model (ADaM) datasets. 
```

Comment:
A metadata structure for clinical data analysis
    and reporting based on Analysis Data Model (ADaM) datasets. 
     The package simplify clinical analysis and reporting tool development
 by defining standardized input, output and workflow. For example, 
    the package can be used to create analysis and reporting planning grid; 
    mock table, and validated A&R results based on consistent input.